1. File "server.py", line 4, in <module>
    mysql = MySQLConnector(app,'friendsdb')
  File "/home/devon/Desktop/DojoAssignments/Python/mySQL/friends/mysqlconnection.py", line 40, in MySQLConnector
    return MySQLConnection(app, 'friendsdb')
  File "/home/devon/Desktop/DojoAssignments/Python/mySQL/friends/mysqlconnection.py", line 15, in __init__
    DATABASE_URI = "mysql://{}:{}@127.0.0.1:{}/{}".format(config['user'], config['password'], config['port'], config['database'])
KeyError: 'database'

Issues: database not defined in config

2. OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'devon'@'localhost' (using password: YES)")
 Issue: User not defined user: "", should be user:"root"

 3 OperationalError: (_mysql_exceptions.OperationalError) (1045, "Access denied for user 'root'@'localhost' (using password: YES)")
 Issue: using incorrect password

4.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (111)")
Issue: incorrect port number
